ft_client/freqtrade_client/ft_rest_client.py

```diff
@@ -484,3 +484,17 @@ def health(self):
         :return: json object
         """
         return self._get("health")
+
+    def list_custom_data(self, trade_id=None, key=None):
+        """Lists custom_data of the running bot.
+
+        :param tradeid: Optional keyword argument - Id of the trade
+
+        :return: json object
+        """
+        params = {}
+        trade_id = -1
+        if trade_id is not None:
+            params["trade_id"] = trade_id
+
+        return self._get("trades/{tradeid}/custom_data", params=params)
```

Comment:
I'd strongly suggest to manually test this. this way - it's not working (for several reasons i guess)

Reply:
This is the last remaining point, I will continue with those manual tests tomorrow.

Reply:
Tested today and both command:

`freqtrade-client --config config.json list_custom_data 1`
`freqtrade-client --config config.json list_open_trades_custom_data`

Are working properly